server/src/test/java/io/spine/examples/kanban/server/board/BoardTest.java

```diff
@@ -129,6 +131,138 @@ void rejection() {
         }
     }
 
+    @Nested
+    class MoveColumns {
+
+        @Test
+        @DisplayName("move columns left")
+        void moveLeft() {
+            ColumnPosition from = ColumnPositions.of(1, 4);
+            ColumnId column = currentBoardState().getColumn(from.zeroBasedIndex());
+            ColumnPosition to = ColumnPositions.of(4, 4);
+            moveColumn(column, from, to);
+            ColumnId actual = currentBoardState().getColumn(to.zeroBasedIndex());
+            assertThat(column).isEqualTo(actual);
+        }
+
+        private Board currentBoardState() {
+            return (Board) context()
+                    .assertEntityWithState(board(), Board.class)
+                    .actual()
+                    .state();
+        }
+
+        private void moveColumn(ColumnId column, ColumnPosition from, ColumnPosition to) {
+            MoveColumn command =
+                    MoveColumn.newBuilder()
+                              .setColumn(column)
+                              .setBoard(board())
+                              .setFrom(from)
+                              .setTo(to)
+                              .vBuild();
+
```

Comment:
This line is redundant.

When leaving a line like this, please consider whether the previous lines are already formatted in a way, that a lot of "breathing room" is left. In this case, the alignment of Proto builder chain allows enough room for readers.